### DISCUSSION TITLE
number slider mushroom card

### DISCUSSION THREAD

**Discussion by cynster** [piitaya/lovelace-mushroom#1033] [Ideas]
I'd love to be able to use the mushroom styled slider used in lights/fan/cover cards for number entities!
See for example the [Sonos](https://www.home-assistant.io/integrations/sonos/) number entities since the default HA-style is very ugly: [Screenshot](https://i.imgur.com/wgi2Ehe.png)

**Comment by smartqasa** [comment]
Slider V2 under the in the My Cards repository https://github.com/AnthonMS/my-cards is the best option but there are several bugs (color temp and color). It does not seem that the developer is supporting it very actively.

**Comment by piitaya** [comment]
Hello. It's available in the 2.6.0 release: https://github.com/piitaya/lovelace-mushroom/releases/tag/v2.6.0

**Comment by bobloadmire** [comment]
is there an option to define a step size? not seeing it in the documentation, adding "step: 5" doesn't work.

**Comment by NilsMinten** [comment]
> is there an option to define a step size? not seeing it in the documentation, adding "step: 5" doesn't work.

From the looks of it it's determined by the step size in the entity. In case of an input number you can change this in the input number settings.

**Reply by bobloadmire** [reply]
I'm looking at my number entity, I don't see any options for it to define a step size though

**Reply by NilsMinten** [reply]
![image](https://user-images.githubusercontent.com/46441845/222585292-e97c7a9b-2b22-447d-bb6d-7695d475fe3c.png)

No clue what might be different, this is what I see.

**Comment by bobloadmire** [comment]
![Screenshot_20230302-153707](https://user-images.githubusercontent.com/4238740/222589207-da4f58b8-aa76-42f3-a296-3bffca9c60eb.png)
All I have unfortunately

**Comment by piitaya** [comment]
The `number` entity step can't be changed from the UI because it's managed by the integration that provide it.

If I'm not wrong, you can customize it in your `configuration.yaml`

```yaml
homeassistant:
  customize:
    number.charge_limit:
      step: 5
```

**Comment by guevara777** [comment]
I do use an input_number-entity and have the abitlity to change stepsize. I´m german and we in germany use "," to seperate full numbers from decimal places. I´ve stepsize set to "0,01" but when i klick the minus oder plus-buttons on the new input-number-card the value always increases by 0,1 instead of 0,01. Is it a bug?

![image](https://user-images.githubusercontent.com/29648553/223468911-dfc062de-6d4a-4a0a-9f9c-7f21eb5aab31.png)

if i use the normal "entities-card" it respects the stepsize correct
![image](https://user-images.githubusercontent.com/29648553/223469435-18a44aba-8031-4683-bbf9-261e9f64735a.png)
